docs/clustertriggerbindings.md

```diff
@@ -6,7 +6,6 @@ designed to encourage reusability clusterwide. You can reference a
 ClusterTriggerBinding in any EventListener in any namespace.
 
 <!-- FILE: examples/clustertriggerbindings/clustertriggerbinding.yaml -->
-
```

Comment:
this stemmed from running `hack/update-docs.sh`

Reply:
yeah, there is some weird new line behavior between `./hack/update-docs.sh` and the `prettier` bot